docs/tutorial/rpc.rst

```diff
@@ -59,23 +63,25 @@ The use of :mod:`compas.rpc` is not restricted to COMPAS packages only. ::
 
 Note that Numpy arrays are automatically converted to lists.
 
-
 Supported data types
 ====================
 
-:mod:`compas.rpc` uses JSON serialisation to transfer data between the "client" (your script)
-and the server runnning the selected CPython environment.
+:mod:`compas.rpc` uses JSON serialization to transfer data between the "client"
+(your script) and the server running the selected CPython environment.
 
-All COMPAS objects (primitives, shapes, data structures, ...) support JSON serialisation through
-their ``to_json`` ``from_json`` methods. On a lower level, these methods convert (complex)
-internal data to simple dicts, and vice versa, with ``to_data`` and ``from_data``.
+All COMPAS objects (primitives, shapes, data structures, etc.) support JSON
+serialization through their ``to_json`` ``from_json`` methods. On a lower level,
+these methods convert (complex) internal data to simple dictionaries, and
+vice versa, with ``to_data`` and ``from_data``.
 
-In combination with custom JSON encoders and decoders this allows for COMPAS objects to be
-serialised and unserialised without loss of information on either side of the RPC communication network.
+In combination with custom JSON encoders and decoders this allows for COMPAS
+objects to be serialized and de-serialized without loss of information on either
+side of the RPC communication network.
 
-Therefore the data types supported by :mod:`compas.rpc` include all native Python data types and COMPAS objects.
-Numpy arrays are automatically converted to lists.
+Therefore the data types supported by :mod:`compas.rpc` include all native Python
+data types and COMPAS objects. Numpy arrays are automatically converted to lists.
 
+linear algebra functions.
```

Comment:
This doesn't belong here.

Reply:
it seems to have appeared out of thin air :)

Reply:
sorry! some merge leftover I guess.... fixed now